### ISSUE TITLE
Hello, I have a question about the main request

### ISSUE THREAD

**Issue by sloba88** [Sujuwa/challenge#1]
"as well as the nearest N number of towns in population"

What does this exactly mean? To show other towns that have similar population?

**Comment by juhah** [comment]
Yep, so if the user searches for Tampere and let's say it has a population of 250 000, then it would show Tampere and for example 5 other towns whose population is less than that of Tampere. Or, a  few under and a few over - the implementation is left open on purpose :)
